Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml

```diff
@@ -40,6 +40,19 @@
       - cargo.funplushies
       - cargo.funartsupplies
       - cargo.funinstruments
+      - livestockbee
```

Comment:
Format these IDs correctly.

Reply:
This is how @Zumorica now wants them. Gone with the cargo prefix. Was going to do the rest at some point.